src/components/editor/inputs/InputLang.jsx

```diff
@@ -17,10 +17,10 @@ import { selectLanguages, hasLanguages } from "selectors/languages"
  * See ISO 639 for the list of registered language codes
  */
 const InputLang = (props) => {
-  const [lang, setLang] = useState(props.lang || "")
-  const langPresent = typeof lang !== "undefined"
-  const [languageSelectorEnabled, setLanguageSelectorEnabled] =
-    useState(langPresent)
+  const [lang, setLang] = useState(props.lang)
+  const [radioButtonValue, setRadioButtonValue] = useState(
+    props.lang === null ? "absent" : "present"
+  )
```

Comment:
this lets us keep track of the selected radio button separately from the existence of the `lang` value and makes the component code below easier to understand ... the default value is based on the existence of a language selection coming in though

Reply:
this makes sense.  I'm wondering if there is a better name for radioButtonValue so it can be clear as a boolean.  `languageIsSelected` ?  `langIsSelectedRadioButton` ?  `langRadioSelector` ?   dunno if it's worth it, but "present" and "absent" are unusual values for a boolean switch.

Reply:
yeah, present and absent are unusual choices, maybe renaming those values to something like "langSelected" (instead of "present") and "noLangSelected" (instead of "absent") will make it clear what is happening -- note this state value is not a boolean, it's the string value of the radio button we will use below to decide which one to check, which is why I called it "radioButtonValue"

Reply:
created constants for those radio button values and then used the better named constants through the code, also makes it more DRY and avoids a case where a typo in a string radio button value that is sprinkled throughout causes things to break